memory/libhugetlbfs.py

```diff
@@ -1,5 +1,28 @@
 #!/usr/bin/env python
 
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Santhosh G <santhog4@linux.vnet.ibm.com>
+#
+# Based on code by:
+# Author: Yao Fei Zhu <walkinair@cn.ibm.com>
+# copyright : 2006 IBM
+# Author: Ashwin Ganti <aganti@google.com>
+# copyright : 2008 Google, Inc.
+# Author: John Admanski <jadmanski@google.com>
+# copyright : 2008 Google, Inc.
```

Comment:
I don't understand the reason for those two. I don't see them in `control` file, nor I see some really significant updates from them, only few updates related to new versions, cleanups, style fixes, runner-related-tweaks, backported-patches-from-newer-versions,

So the question is why to include those and not to include the other guys, who did similar contributions like Lucas, Martin, or Julius? From what I understand about GPL only the original author and all authors who claimed their contribuiton (wrote themselves in the license, which is for autotest the `control` file) should be added.

I mean yes, if those guys contacted you and said they did major contribution, then fine, we can include them, but they need to claim the contribution, which they so far did not.